Source/OrchBase/P_builder.cpp

```diff
@@ -1394,16 +1401,19 @@ unsigned P_builder::WriteThreadVars(string& task_dir, unsigned coreNum,
           edgeInitialiser << (*device)->idx << ",";                             // tgt
           edgeInitialiser << p_edge->second.iArc->second.fr_n->first << ",";    // src
           //====================================================================
-          
+	  
           // if the pin has properties,
-          if (p_edge->second.data->pP_pintyp->PinPropsSize)
+	  // if (p_edge->second.data->pP_pintyp->PinPropsSize)

```

Comment:
Can this line be removed rather than left commented? We have version control if we need it again.

Reply:
I just left it in there in case there was further comment from anyone. Removed.